tests/test_evaluation/test_metrics/test_multi_task_metrics.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+
+from mmcls.evaluation.metrics import (MultiTasksMetric, Accuracy, SingleLabelMetric)
+from mmcls.structures import MultiTaskDataSample
+
+
+class MultiTaskMetric(TestCase):
+
+    pred = [
+        MultiTaskDataSample().set_pred_task(i).set_gt_task(
+            k).to_dict() for i,  k in zip([
+                {
+                    'task0': torch.tensor([0.7, 0.0, 0.3]),
+                    'task1': torch.tensor([0.5, 0.2, 0.3])
+                },
+                {
+                    'task0': torch.tensor([0.0, 0.0, 1.0]),
+                    'task1': torch.tensor([0.0, 0.0, 1.0])
+                },
+            ], [{'task0': 0, 'task1': 2}, {'task0': 2, 'task1': 1}])
+    ]
+
+    task_metrics = {
+            'task0': [dict(type='Accuracy', topk=(1, ))],
+            'task1': [
+                dict(type='Accuracy', topk=(1, 3)),
+                dict(type='SingleLabelMetric', items=['precision', 'recall'])
+                ]
+            }
+
+    def test_init(self):
+        metrics = MultiTasksMetric(self.task_metrics)
+        self.assertIsInstance(metrics.Accuracy_task0, Accuracy)
```

Comment:
* I won't put it in the root metric object
* Better to name it task0_<something>
* Accuracy is strange to have in the key here
* it should be a list of metrics for each task